var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -41,6 +41,9 @@ class Ncurses(AutotoolsPackage):
     variant('symlinks', default=False,
             description='Enables symlinks. Needed on AFS filesystem.')
 
+    variant('wide', default=True,
```

Comment:
I may be wrong, but I don't think the concretizer has any semblance of ADDING or SUBTRACTING. `+wide` and `~wide` are equivalent to `wide=True` and `wide=False`; multi-valued variants aren't anything special. The concretizer doesn't have any problems with tildes in particular, the problem arises when a spec differs from the default value and the package is far down in the DAG. Spack doesn't backtrack far enough to resolve these conflicts. So a multi-valued variant wouldn't change anything. @tgamblin can correct me if I'm wrong.